README.md

```diff
@@ -9,7 +9,7 @@ This is the official Pipedrive API wrapper-client for NodeJS based apps, distrib
 # Install
 
 ```
-npm install pipedrive
```

Comment:
This refers to https://www.npmjs.com/package/pipedrive
so its supposed to include package name, you can't just remove it